src/components/organisms/PageHeader/ProjectSelection.styled.tsx

```diff
@@ -112,7 +112,6 @@ export const ProjectsMenuContainer = styled.div`
 `;
 
 export const ProjectTableActions = styled.div`
-  display: flex;
   align-items: center;
   justify-content: center;
```

Comment:
Shouldn't you remove `align-items` and `justify-content` as well?

Reply:
I wasn't sure about that its against css standarts or not, I had two options;
-remove display flex and add display flex to hovered state's css
or
-remove that 3 lines and add to hovered state's css

which one do you think should I prefer?